src/ILLink.Tasks/LinkTask.cs

```diff
@@ -303,6 +316,13 @@ protected override string GenerateResponseFileCommands ()
 			if (OutputDirectory != null)
 				args.Append ("-out ").AppendLine (Quote (OutputDirectory.ItemSpec));
 
+			if (NoWarn != null)
+				args.Append ($"--nowarn \"{NoWarn}\"");
+
+			if (_trimmerAnalysisWarnings is bool trimmerAnalysisWarnings &&
+				trimmerAnalysisWarnings == true)
+				args.Append ("--nowarn 2006;2026;2037;2043");
```

Comment:
The command line parser for linker below doesn't handle multiple `--nowarn` options, the last one will win. So the user won't be able to combine the analysis warning disable with any other nowarn.

Reply:
I think the warnings should be disabled from MSBuild [targets](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L99) until we properly define the categories. It still might be nice to allow multiple `--nowarn` options.